### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by EdwardTown1** [ycba-cia/blacklight-collections2#228]
@yulgit1 @edgartdata  Hi both - Beth Miller commented via email to @robl about the lack of legibility of the text on the carousel and I that it may well be worthwhile reaching out to Mike Vaughan in Accessibility to ask for his opinion. I can see what she means –there is an issue there with painter's set. Something for us to discuss in our weekly meeting on Friday. 

Very best, 
Ed

**Comment by edgartdata** [comment]
Invite Mike Vaughn to comment on the carousel and take a look at the rest of the catalog. Rob to contact him.

**Comment by robl** [comment]
_Emailed Mike Vaughan as below:_

> Also, I know that your team advised earlier (before my time at Yale?) on the YCBA Collections Search site that Eric James (cc'ed here) built and manages on Blacklight. Now that we're post-launch there as well, a few aspects of that site have changed since you last had a look, I believe.
> 
> The main thing in this regard is an image carousel that was added shortly before launch, and which may have things to improve for accessibility (white text-overlay contrast on certain carousel images, ability to stop motion, etc.) if the JS tool it runs on allows. If you could please have a look and let us know, that would be great. Because the collections search site was developed in-house on an existing open-source platform, we don't have the ability to push a vendor to fix things under warranty like we do for the main site; but we look forward to improving its accessibility as best we can within the affordances and constraints of the software it uses. The site is at:
> 
> https://collections.britishart.yale.edu/
> 
> And examples of two other key types of page on it are:
> 
> https://collections.britishart.yale.edu/?f%5Bcollection_ss%5D%5B%5D=Paintings+and+Sculpture
> 
> https://collections.britishart.yale.edu/catalog/tms:34
> 
> You'll see that the destination URL for the "Accessibility at Yale" link in the footer has not been updated to match the corrected URL in the Drupal site's footer, which that Blacklight footer emulates; I just made a github issue for this, so it's known and in the queue for fixing.
> 
> many thanks,
> Rob

**Comment by yulgit1** [comment]
One critical issue: "The carousel on the page is automatically updating. Anything on a page that is moving, blinking, scrolling, or auto-updating should have a mechanism to allow a user to pause, stop, or hide such movement."

The carousel actually does stop on hover, but the fact this wasn't noticed might be a problem in itself.  I'm going to turn off the movement.

**Comment by robl** [comment]
Thanks, @yulgit1 . Yes, I believe the stop or pause function needs to be exposed in a visible way. Stopping movement for now sounds perfect, and perhaps we can see whether a pause/resume button or other interface element could eventually be added to trigger the same behavior that is now triggered by hover location. If so, we could then turn movement back on by default, and clicking/tapping on that new interface element or hovering could stop or start movement. In principle.
